tf/global/global.tf

```diff
@@ -0,0 +1,61 @@
+resource "azurerm_resource_group" "global-rg" {
+  name     = "sbd-global"
+  location = "South Central US"
+}
+
+resource "azurerm_key_vault" "global-kv" {
+  name                        = "sbdvault"
+  location                    = azurerm_resource_group.global-rg.location
+  resource_group_name         = azurerm_resource_group.global-rg.name
+  enabled_for_disk_encryption = true
+  tenant_id                   = "e86183dc-d7cc-4132-8b39-a8de37272433"
+
+  sku_name = "premium"
+
+  network_acls {
+    default_action = "Deny"
+    bypass         = "AzureServices"
+  }
+}
+
+resource "azurerm_key_vault_access_policy" "test" {
+  key_vault_id = azurerm_key_vault.global-kv.id
+
+  tenant_id = "e86183dc-d7cc-4132-8b39-a8de37272433"
```

Comment:
This is our hardcoded AAD tenant.  I don't think we have a way to automatically provision one, so we'll have to provide instructions and parameterize this.